tests/_data/invalid/manifest/en_IN.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Why do you need this? It is a valid one and exists in `invalid` directory.

Reply:
I have to check the error code so I create it inside the invalid directory. I removed it now.